addOns/ascanrulesAlpha/src/test/java/org/zaproxy/zap/extension/ascanrulesAlpha/HiddenFilesScanRuleUnitTest.java

```diff
@@ -114,7 +119,50 @@ public void shouldRaiseAlertIfTestedUrlRespondsOkWithRelevantContent()
     }
 
     @Test
-    public void shouldRaiseAlertWithLowConfidenceIfTestedUrlRespondsForbidden()
+    public void shouldRaiseAlertIfTestedUrlRespondsOkWithCustomPayload()
```

Comment:
The handling is slightly different but maybe this isn't necessary given `shouldRaiseAlertWithLowConfidenceIfTestedUrlRespondsOkToCustomPayload`